samples/qa-virtualgridlist/src/components/LocaleSwitch/LocaleSwitch.js

```diff
@@ -4,12 +4,12 @@ import ToggleButton from '@enact/moonstone/ToggleButton';
 
 const LocaleSwitch = (props) => {
 	const {rtl, updateLocale} = useI18nContext();
-	const onClick = useCallback(() => {
+	const onToggle = useCallback(() => {
 		updateLocale(!rtl ? 'ar-SA' : 'en-US');
 	}, [rtl, updateLocale]);
 
 	return (
-		<ToggleButton onClick={onClick} {...props}>RTL</ToggleButton>
+		<ToggleButton onToggle={onToggle} {...props}>RTL</ToggleButton>
```

Comment:
I'm seeing `<ToggleButton onClick .... />` in qa-a11y, qa-scroller, and qa-scrollernative also.
Please fix them all.

Reply:
Done.